enterprise/frontend/src/metabase-enterprise/whitelabel/components/SwitchWidget/SwitchWidget.tsx

```diff
@@ -0,0 +1,27 @@
+import { Switch } from "metabase/ui";
+import type { SwitchProps } from "metabase/ui";
+
+interface MetabaseBooleanSetting {
+  value: boolean | null;
+  default: boolean;
+}
+
+type SwitchWidgetProps = SwitchProps & {
+  setting: MetabaseBooleanSetting;
+  onChange: (value: boolean) => void;
+};
+
+export function SwitchWidget({
```

Comment:
This is a new component for the new design.
![image](https://github.com/metabase/metabase/assets/1937582/d63d521c-66da-49b2-83a6-70ac3a79edf2)